src/utils/email.js

```diff
@@ -24,6 +24,4 @@ const send = async ({ email, id, name, position }) => {
   });
 };
 
-export default {
-  send,
-};
+export default send;
```

Comment:
this needs to be reflected in the actual code as well

Reply:
gotcha --- I looked at the place where its used (there's only one), its already importing it correctly like so: 
![image](https://github.com/acm-ucr/hackathon-website/assets/64669569/074d84de-7fb9-42fd-b641-5eb22b4b61f0)

Reply:
But its used as EmailSerivce.send(), it would just become send

Reply:
Got it, I replaced them all with send.